src/cache/factory.py

```diff
@@ -47,7 +47,7 @@ def is_cache_available() -> bool:
         use geo-coding.
     """
     env = CacheEnvReader()
-    if env.redis_host():
+    if env.redis_host() and env.cache_queries():
```

Comment:
this change is already in #449, so perhaps we should merge #449 first and then double check what happens when rebasing, i.e. make sure nothing gets undone!